main.py

```diff
@@ -140,8 +140,10 @@ def save_file():
 
 root = CTk()
 
-screen_width = root.winfo_screenwidth()
-screen_height = root.winfo_screenheight()
+screen_width = 900
+screen_height = 600
+# screen_width = root.winfo_screenwidth()
```

Comment:
Nice, simple change.

Reply:
This 900/600 change was just for dev testing, you may want to change it back to fullscreen for the pull request.